.github/workflows/set-milestone-on-referenced-issue.yml

```diff
@@ -0,0 +1,33 @@
+#############################################################################
+# GitHub action to set latest milestone on issue of merged PR.
+#
+#############################################################################
+name: 'Milestone for PR'
```

Comment:
Does this name go on the CI too? If so, it duplicates the other name.

Please confirm and if so, lets keep this name as is and change the other to just the PR number.

Reply:
This is the workflow name. Without it, we would see `.github/workflows/set-milestone-on-referenced-issue.yml` so we need it.
> The name of your workflow. GitHub displays the names of your workflows on your repository's "Actions" tab. If you omit name, GitHub sets it to the workflow file path relative to the root of the repository.

![Screenshot from 2023-03-03 10-05-03](https://user-images.githubusercontent.com/23459549/222680811-e7b28890-5f0e-4bca-9469-990fb446586d.png)

Reply:
The one below is the job name which can be seen once you click on the action:
![Screenshot from 2023-03-03 10-17-05](https://user-images.githubusercontent.com/23459549/222681256-40d224a8-ff98-410d-885f-be643b330511.png)

Reply:
I am not saying to remove it, I am just saying things have to be shorter. Also please show me what it will look like in CI display, not under Actions.

Example:
CI says `Checker / check (nullness-optional-interning)`
It comes from 
https://github.com/checkstyle/checkstyle/blob/master/.github/workflows/checker-framework.yml#L1
and 
https://github.com/checkstyle/checkstyle/blob/master/.github/workflows/checker-framework.yml#L15

So name I mentioned does get put into CI display, and I believe so does job name.
So won't CI display `Milestone for PR / Milestone for PR #####'.

I am just asking to shorten it to display `Milestone for PR / #####`.
So first name can stay the same, but just name will just be the PR number.

Reply:
Ohhhh, got it now. Done.

Reply:
Please include `#` like it was before so we can understand what the number being printed is.